application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -54,11 +54,12 @@ public static void disableButtons(@NotNull Message message) {
      * @return the generated embed
      */
     public static @NotNull MessageEmbed generateEmbed(@Nullable String title,
-            @NotNull CharSequence content, @Nullable User user, @Nullable Color ambientColor) {
+            @NotNull CharSequence content, @Nullable User user, @Nullable Color ambientColor,
+            @Nullable String footer) {
         return new EmbedBuilder().setTitle(title)
             .setDescription(content)
             .setTimestamp(Instant.now())
-            .setFooter(user == null ? null : user.getName())
+            .setFooter(user == null ? null : user.getName() + " " + footer)
```

Comment:
Based on the naming, this is a mistake. The `generateEmbed` method should not decide how to "decorate" the footer but give the caller the full freedom in designing the footer message. Change this line to `setFooter(footer)` and thats it.